services/database/migrations/20231109160022-create-alliance-schema.js

```diff
@@ -17,412 +23,122 @@ exports.setup = function (options, seedLink) {
-      DO
-      $$
-          DECLARE
-              t_name text;
-          BEGIN
-              FOR t_name IN (VALUES ('compiled_contracts'),
-                                    ('verified_contracts'))
-                  LOOP
-                      EXECUTE format('CREATE TRIGGER insert_set_created_by
-                              BEFORE INSERT ON %I
-                                  FOR EACH ROW
-                              EXECUTE FUNCTION trigger_set_created_by()',
-                                     t_name);
-      
-                      EXECUTE format('CREATE TRIGGER insert_set_updated_by
-                              BEFORE INSERT ON %I
-                                  FOR EACH ROW
-                              EXECUTE FUNCTION trigger_set_updated_by()',
-                                     t_name);
-      
-                      EXECUTE format('CREATE TRIGGER update_reuse_created_by
-                              BEFORE UPDATE ON %I
-                                  FOR EACH ROW
-                              EXECUTE FUNCTION trigger_reuse_created_by()',
-                                     t_name);
-      
-                      EXECUTE format('CREATE TRIGGER update_set_updated_by
-                              BEFORE UPDATE ON %I
-                                  FOR EACH ROW
-                              EXECUTE FUNCTION trigger_set_updated_by()',
-                                     t_name);
-                  END LOOP;
-          END;
-      $$ LANGUAGE plpgsql;      
-      
-      
-      `
-      ),
-    ],
-    callback
-  );
+  async.series([db.runSql.bind(db, allianceSchemaSql)], callback);
 };
 
 exports.down = function (db, callback) {
   async.series(
     [
       db.dropTable.bind(db, "verified_contracts"),
+      db.dropTable.bind(db, "compiled_contracts_sources"),
+      db.dropTable.bind(db, "sources"),
       db.dropTable.bind(db, "compiled_contracts"),
       db.dropTable.bind(db, "contract_deployments"),
       db.dropTable.bind(db, "contracts"),
       db.dropTable.bind(db, "code"),
+
       db.runSql.bind(db, 'DROP FUNCTION IF EXISTS "trigger_set_created_at";'),
       db.runSql.bind(db, 'DROP FUNCTION IF EXISTS "trigger_reuse_created_at";'),
       db.runSql.bind(db, 'DROP FUNCTION IF EXISTS "trigger_set_updated_at";'),
       db.runSql.bind(db, 'DROP FUNCTION IF EXISTS "trigger_set_created_by";'),
       db.runSql.bind(db, 'DROP FUNCTION IF EXISTS "trigger_reuse_created_by";'),
       db.runSql.bind(db, 'DROP FUNCTION IF EXISTS "trigger_set_updated_by";'),
+
+      db.runSql.bind(
+        db,
+        'DROP FUNCTION IF EXISTS "validate_runtime_transformations";',
+      ),
+      db.runSql.bind(
+        db,
+        'DROP FUNCTION IF EXISTS "validate_creation_transformations";',
+      ),
+      db.runSql.bind(db, 'DROP FUNCTION IF EXISTS "validate_transformations";'),
+      db.runSql.bind(
+        db,
+        'DROP FUNCTION IF EXISTS "validate_transformations_call_protection";',
```

Comment:
Why were were dropping all these checks again? Maybe we should add a comment here.

Reply:
It's the down function. It should restore the database state before the migration. I can add a comment for that.